src/Components/Patient/PatientHome.tsx

```diff
@@ -368,10 +297,10 @@ export const PatientHome = (props: any) => {
         </div>
       </div>
     );
-  } else if (sampleListData.length > 0) {
+  } else if (sampleListData?.count || 0 > 0) {
```

Comment:
That's hard to read.

```suggestion
  } else if (sampleListData?.count) {
```